configure.ac

```diff
@@ -6,6 +6,7 @@ dnl  General Public License, version 3 or any later version (LGPLv3 or
 dnl  later), or the GNU General Public License, version 2 (GPLv2), in all
 dnl  cases as published by the Free Software Foundation.
 
+// clang-format off
```

Comment:
Why this is needed. _clang-format_ shouldn't touch this file.

Reply:
But it does, unfortunately, at least on the latest `clang-format version 10.0.1 (Fedora 10.0.1-2.fc32)`. Adding this line is the only way I found to not format it.